.eslintrc

```diff
@@ -1,176 +1,176 @@
 {
```

Comment:
I use them all on our main fairly large project and have no need in babel-eslint. Seriously.

Right, will separate indentation into another PR.